README.md

```diff
@@ -83,19 +84,25 @@ cp .env.example .env
 
 #### 5. Set up database schema
```

Comment:
Can you change the heading, now both step 5 and 6 has same heading. I would like to mark the `docker-compose` step as optional if the developers want to spin on a database using Docker or they can use any database they want.